### PR TITLE
Reverting get_previous_version logic to get the first tag_name's value from json

### PR DESCRIPTION
**Pull request by gauravsaini04**
**Features names**:
 
 * Kubectl-helm-minikube
 
 **Description**:
 
 This PR corrects the existing code with the following functionality:
 
 * `kubectl-helm-minikube` feature was picking up the second to latest tag from json array of releases url, now have changed that to pick the first value for `tag_name` key in the same.
 
 _Changelog_:
 
 * Updated `install.sh` file
 * Updated tests
 
 **Checklist**:
 
 * [x]   Checked that applied changes work as expected

### CODE REVIEW

File: src/kubectl-helm-minikube/install.sh [hunk 1]
```diff
@@ -159,7 +159,7 @@ fi
 get_previous_version() {
     repo_url=$1
     # this would del the assets key and then get the second encountered tag_name's value from the filtered array of objects
-    curl -s "$repo_url" | jq -r 'del(.[].assets) | .[1].tag_name' 
+    curl -s "$repo_url" | jq -r 'del(.[].assets) | .[0].tag_name'
```
Comment (samruddhikhandale) [inline src/kubectl-helm-minikube/install.sh:162]: Can we bump patch version of this Feature?

As a rule of thumb, whenever you contribute changes to any Feature, you always need to bump the Feature versions. Follow semantic versioning rules to determine the version bump.

Failing the bump version will lead into not releasing a new version of the Feature with these changes.

File: src/kubectl-helm-minikube/install.sh [hunk 2]
```diff
@@ -159,7 +159,7 @@ fi
 get_previous_version() {
     repo_url=$1
     # this would del the assets key and then get the second encountered tag_name's value from the filtered array of objects
```
Comment (samruddhikhandale) [inline src/kubectl-helm-minikube/install.sh:161]: Can we update the comment?

File: test/docker-in-docker/docker_build_fallback_buildx.sh [hunk 3]
```diff
@@ -26,7 +26,7 @@ get_latest_version() {
 
 # Function to fetch the previous version of the plugin
 get_previous_version() {
-    curl -s "$repo_url" | jq -r 'del(.[].assets) | .[1].tag_name' # this would del the assets key and then get the second encountered tag_name's value from the filtered array of objects
+    curl -s "$repo_url" | jq -r 'del(.[].assets) | .[0].tag_name' # this would del the assets key and then get the second encountered tag_name's value from the filtered array of objects
```
Comment (samruddhikhandale) [inline test/docker-in-docker/docker_build_fallback_buildx.sh:29]: Same here. Can we update the comment?

File: src/kubectl-helm-minikube/install.sh (hunk 1 above)
Comment (gauravsaini04) [reply src/kubectl-helm-minikube/install.sh:162]: made changes to the file

File: test/docker-in-docker/docker_build_fallback_buildx.sh (hunk 3 above)
Comment (gauravsaini04) [reply test/docker-in-docker/docker_build_fallback_buildx.sh]: updated the comment

File: src/kubectl-helm-minikube/install.sh (hunk 2 above)
Comment (gauravsaini04) [reply src/kubectl-helm-minikube/install.sh]: updated the comment 👍

**Review by samruddhikhandale — APPROVED** [review]
Nice 🚀

**Pull request merged**